blocks/v2-feature-carousel/v2-feature-carousel.js

```diff
@@ -0,0 +1,18 @@
+import { createElement, removeEmptyTags } from '../../scripts/scripts.js';
+
+export default function decorate(block) {
```

Comment:
Can we use a constant for the block name? So it can be reused in all the JS file?
`const blockName = 'v2-feature-carousel';
 el.classList.add('${blockName}__card');
`

Reply:
Done